_posts/2021-06-22-solidity-0.8.6-release-announcement.md

```diff
@@ -27,9 +27,10 @@ short changelog.
 ### Optimizer Setting
 
 We discovered that the default optimizer setting differed depending on the way
-the compiler was accessed: Using the compiler on the commandline ("``solc``")
+the compiler was invoked: Using the compiler on the commandline ("``solc``")
 caused minimal optimizer features to be activated (the intended functionality)
-while using ``solc-js`` or Standard JSON all optimizer features were disabled.
+while using ``solc-js`` all optimizer features were disabled. This behavior
+could also be triggered in Standard JSON mode.
```

Comment:
I would remove stardard json from here, because this paragraph starts with "default optimizers setting differed", and I would not necessarily call `optimizer.settings = {}` a "default".

Reply:
ok. Remove it completely?

Reply:
Removed.